gensim/models/experimental/drmm.py

```diff
@@ -0,0 +1,60 @@
+import keras
+from keras.models import Model
+from keras.layers import Input, Embedding, Dense, Lambda, Dropout, Activation, Reshape, Dot, Permute  # noqa
+from keras.activations import softmax
+
+
+class DRMM:
```

Comment:
General comment about ALL networks

**Documentation:**
- missing (or bad) documentation
- no usage examples (see https://docs.python.org/3/library/doctest.html)
- no links to paper/information resourse about this network

**Interfaces:**
- Still no methods for apply network to new data.
- Missing save/load functionality

Reply:
> Still no methods for apply network to new data.
This is there
Please refer to the Ipython notebook.

Rest, I'll work on.